examples/2d/elixir_euler_unstructured_quad_free_stream.jl

```diff
@@ -0,0 +1,61 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_constant
+boundary_conditions = BoundaryConditionDirichlet(initial_condition)
+
+###############################################################################
+# Get the DG approximation space
+
+polydeg = 6
+surface_flux = flux_hll
+solver = DGSEM(polydeg, surface_flux)
```

Comment:
```suggestion
solver = DGSEM(polydeg=6, surface_flux=flux_hll)
```
We added these keyword arguments some time ago to make code like this easier to read (in a single line).

Reply:
Done, I also made the `periodicity`a keyword for the unstructured mesh to improve readability there too